index.html

```diff
@@ -5,8 +5,11 @@
     <title>Jonathan Daniel Web Devloper</title>
```

Comment:
I see your separate HTML docs here in the PR. Consolidate all of that code into one HTML doc, and then we can work with you on implementing the 'tab-based' navigation, like we're doing in the blog.